### ISSUE TITLE
Signed releases

### ISSUE THREAD

**Issue by shibumi** [google/xsecurelock#77]
Hi,
It would be nice to have signed releases for the most secure screen locker ;)

**Comment by divVerent** [comment]
Can you point me to a standard for that? As currently I rely on autoconf (make dist), which doesn't do that.

Or do you merely mean separate GPG signatures?

**Comment by shibumi** [comment]
Yes, I mean seperate GPG signatures. Most linux distributions obtain your software via the github release function.. if you would provide a detached GPG signature, we could check if the release is really from you and not from somebody hostile who hijacked your github repository.

Well, just generate a key. It's still better as without.
